opba-auth-rest-impl/src/main/java/de/adorsys/opba/tppauthapi/config/AuthConfig.java

```diff
@@ -0,0 +1,35 @@
+package de.adorsys.opba.tppauthapi.config;
+
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.crypto.RSASSASigner;
+import lombok.SneakyThrows;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.security.KeyFactory;
+import java.security.PrivateKey;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.util.Base64;
+
+@Configuration
+public class AuthConfig {
+
+    @Bean
+    JWSHeader jwsHeaderBuilder(TppProperties tppProperties) {
+        return new JWSHeader.Builder(JWSAlgorithm.parse(tppProperties.getJwsAlgo())).build();
+    }
+
+    @Bean
+    RSASSASigner rsassaSigner(TppProperties tppProperties) {
```

Comment:
Do not rely on concrete classes - signature should be JWSSigner, this allows you to use i.e. ECDSASigner if Quantum computers will rule the world.

Reply:
done